lib/pure/random.nim

```diff
@@ -220,7 +220,8 @@ proc rand*(r: var Rand; max: Natural): int {.benign.} =
   ## * `rand proc<#rand,int>`_ that returns an integer using the default
   ##   random number generator
   ## * `rand proc<#rand,Rand,range[]>`_ that returns a float
-  ## * `rand proc<#rand,Rand,HSlice[T,T]>`_ that accepts a slice
+  ## * `rand proc<#rand,Rand,HSlice[T:+Ordinal+or+float+or+float32+or+float64,T:+Ordinal+or+float+or+float32+or+float64]>`_
```

Comment:
Oh, you're right. I had used the URLs of the links in the left menu which are actually also broken. I updated the links in the body of the page to be correct now (replacing the plus signs with spaces), and I also fixed the other HSlice links I had missed. And since I'm not sure how to fix those broken left menu links, I added a note about them here: https://github.com/nim-lang/Nim/issues/16337